packages/tools/src/types/ToolSpecificAnnotationTypes.ts

```diff
@@ -173,3 +173,35 @@ export interface RectangleROIStartEndThresholdAnnotation extends Annotation {
     };
   };
 }
+
+export interface PlanarFreehandROIAnnotation extends Annotation {
+  metadata: {
+    cameraPosition?: Types.Point3;
+    cameraFocalPoint?: Types.Point3;
+    viewPlaneNormal?: Types.Point3;
+    viewUp?: Types.Point3;
+    annotationUID?: string;
+    FrameOfReferenceUID: string;
+    referencedImageId?: string;
+    toolName: string;
+  };
+  data: {
+    handles: {
+      points: Types.Point3[];
+      activeHandleIndex: number | null;
+      textBox: {
+        hasMoved: boolean;
+        worldPosition: Types.Point3;
+        worldBoundingBox: {
+          topLeft: Types.Point3;
+          topRight: Types.Point3;
+          bottomLeft: Types.Point3;
+          bottomRight: Types.Point3;
+        };
```

Comment:
It doesn't yet as that isn't in scope for my client's piece of work at this moment. I believe. However its here because it inherits from Annotation, and it would be more painful to make it not inherit and be different from everything else for now. I'll add a comment, I think it will come in a future PR at some point.